common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -106,10 +106,11 @@ public TachyonConf() {
 
   /**
    * Test constructor for TachyonConfTest class.
-   *
+   * 
```

Comment:
Please trim the trailing whitespace. (Applies to other areas of the code as well)